src/gg.regression.unity.bots/Editor/Scripts/ActionManager/RGActionAnalysis.cs

```diff
@@ -253,7 +257,7 @@ private IEnumerable<ExpressionSyntax> FindCandidateValuesForLocalVariable(ILocal
         /// </summary>
         private IEnumerable<SyntaxNode> FindDerivedUserInput(ExpressionSyntax expr)
         {
-            const int maxDepth = 5;
+            const int maxDepth = 10;
```

Comment:
This change (and the other one in this method) is to fix a bug I found with the analysis of raycasts when checking raycast from the new Input System's mouse position.